client/me/billing-history/receipt.jsx

```diff
@@ -126,6 +130,36 @@ const BillingReceipt = React.createClass( {
 		);
 	},
 
+	renderTitle() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<QueryBillingTransactions />
+				<HeaderCake backHref={ purchasesPaths.billingHistory() }>
+					{ translate( 'Billing History' ) }
+				</HeaderCake>
+			</div>
+		);
+	},
+
+	renderPlaceholder() {
+		const { translate } = this.props;
+
+		return (
+			<Main>
+				{ this.renderTitle() }
+				<Card compact className="billing-history__receipt-card">
+					<div className="billing-history__receipt">
+						<div className="billing-history__receipt-loading">
+							{ translate( 'Loading…' ) }
```

Comment:
This works, though we tend to use a pulsing block loader. (cc @mtias for suggestions) We have a sass [mixin](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/mixins/_placeholder.scss) for this, which you can add to your styles with: 
```scss
.component__placeholder {
     @include placeholder();
}
```

Feel free to update this in a follow up PR though, as prod currently breaks in this receipt view with a clean cache (localStorage cleared).